service/controller/aws/v14/resource/kubeconfig/testdata.go

```diff
@@ -0,0 +1,24 @@
+package kubeconfig
```

Comment:
Let's just have this as a constant in the desired state test. No need for a separate file.